src/script/main/app.js

```diff
@@ -852,13 +852,14 @@ class App {
 
 $(() => {
   enableLogging(Config.FEATURE.ENABLE_DEBUG);
-  if (document.getElementById('wire-main-app')) {
+  const appContainer = document.getElementById('wire-main');
+  if (appContainer) {
```

Comment:
The test is slightly different (not on the same DOM element), but the 2 elements have the exact same semantic.